_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,336 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [text node][] that is [visible][] and a [child][] in the [flat tree][] of an HTML element, whose content changes through an action that was not initiated by the user, anytime after the [readiness][document readiness] of the document the [text node][] belongs to is equal to "complete".
+
+## Expectation 1
+
+For each test target a [mechanism][] is provided to pause or stop the updating of the content of the [text node][].
```

Comment:
Also why not just do `[my-definition](link-to-my-definition)`, as against enumerating it all at the bottom of the time.

This pollutes the `spelling-ignore.yml` file to have keywords that are not necessarily english dictionary words, but made up hypenated words.

Reply:
Isn't that what the spelling-ignore is for? Anyway, if this is up for discussion, a new issue should be opened.